### PR TITLE
endpoint and test added. Added class in server file.

### PR DESCRIPTION
**Pull request by 100rabhpawar**

### CODE REVIEW

File: src/models/types/questions.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { artistBudget } from './artist-budget';
+import { targetAudience } from './target-audience';
+import { whatSellsMost } from './what-sells-most';
+
+export type Questions = {
+
+        ueserId: String;
```
Comment (sagargudka) [inline src/models/types/questions.ts:7]: userId

File: src/models/types/questions.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        formName: String;
+        eventType: String;
+        venue: Array<String>;
+        artisBudget: artistBudget;
```
Comment (sagargudka) [inline src/models/types/questions.ts:11]: artistBudget

File: src/server.ts [hunk 3]
```diff
@@ -11,23 +11,33 @@ import ArtistRoute from './artist-routes';
...
 
-export default app;
+    this.app.use(bodyParser.json());
+    this.app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDoc));
+
+    this.app.use('/artists', new ArtistRoute(this.artistService).router);
+  }
+}
+
+// export default app;
```
Comment (sagargudka) [inline src/server.ts:43]: remove unneeded code

File: src/services/artist.ts [hunk 4]
```diff
@@ -24,4 +25,46 @@ export default class ArtistService implements ArtistServiceInteface {
   getRecommendation(id: string): ArtistRecommendation[] {
     return this.artistRecommendationRepo.getArtistRecommendations(id);
   }
+
+  createNewRecommendation(questions: Questions): String {
+    const artistRecommendation = {
```
Comment (sagargudka) [inline src/services/artist.ts:30]: the response is not supposed to be hardcoded in the function. Place it in in a separate json file the way it is has been done in the endpoints and operations I added

File: src/services/artist.ts [hunk 5]
```diff
@@ -24,4 +25,46 @@ export default class ArtistService implements ArtistServiceInteface {
...
+        },
+      },
+      summary: 'adele is well known bollywod singer with lot of hits',
+    };
+
+    this.artistRecommendationRepo.addNewRecommendation(artistRecommendation);
+    const questionId = '1234';
+    return questionId;
+  }
 }
```
Comment (sagargudka) [inline src/services/artist.ts:70]: where is the application level tests for the route that hits this method

File: src/models/types/what-sells-most.ts [hunk 6]
```diff
@@ -0,0 +1,5 @@
+export type whatSellsMost = {
```
Comment (rebaseteams) [inline src/models/types/what-sells-most.ts:1]: these must be Array of brands

File: src/models/types/target-audience.ts [hunk 7]
```diff
@@ -0,0 +1,5 @@
+export type targetAudience = {
+    ageGroup: Array<String>;
+    genre: Array<String>;
```
Comment (rebaseteams) [inline src/models/types/target-audience.ts:3]: Genre should be a separate type each genre will have its own id

**Comment by rebaseteams** [comment]
There are conflicts in the branch.

**Pull request closed without merging**